index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    
+    <header class="toolbar">
+     
+       <nav class="logo"><a  href="#"><img src="Images\Logo Placeholder.png" alt="logo" /></a></nav> 
+      <img class="icons" src="Images\Enabled.png" alt="threeline" />
+  </header>
+
+  <section class="sec">
+    <div class="headline">
+    
```

Comment:
- [ ] Great job using a section tag for the headline! 🚀 However, please make sure that every tag is closed! At the moment you don't have a closing `</section>` tag! 🤓